### PR TITLE
Panning

### PR DESCRIPTION
**Pull request by JaspervanTilburg**

### CODE REVIEW

File: src/main/java/gui/MenuController.java [hunk 1]
```diff
@@ -97,8 +97,8 @@ public void initialize() {
 
         recentController.initialize(prefs);
         ps = new PrintStream(new Console(consoleArea));
-        System.setErr(ps);
-        System.setOut(ps);
+        //System.setErr(ps);
+        //System.setOut(ps);
```
Comment (LBoleij) [inline src/main/java/gui/MenuController.java]: Do we want this in master so we don't have to use our own console anymore?

**Comment by Jipje** [comment]
A slight bug when using bookmarks. The size of the panning button doesn't udpdate when using bookmarks.

File: src/main/java/gui/sub_controllers/BookmarkController.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ public void loadBookmarks(String stringOfFile) {
         // And that file has no bookmarks
         // I want to not see the old bookmarks.
         // Initializing this number as -2 ensures the above user story.
-        for (int i = -2; i <= largestIndex; i++) {
+        for (int i = -2; i <= largestIndex; i++) { //TODO magic number
```
Comment (Jipje) [inline src/main/java/gui/sub_controllers/BookmarkController.java]: The magic number is described just above here...

File: src/main/java/gui/sub_controllers/ZoomController.java [hunk 3]
```diff
@@ -12,30 +12,43 @@
  */
 public class ZoomController {
 
+    private static final double BUTTON_ZOOM_IN_FACTOR = 0.8;
+    private static final double BUTTON_ZOOM_OUT_FACTOR = 1.25;
```
Comment (Jipje) [inline src/main/java/gui/sub_controllers/ZoomController.java]: This is tidy. Nice.

File: src/main/java/gui/GraphDrawer.java [hunk 4]
```diff
@@ -337,5 +345,9 @@ public double getxDifference() {
     public int mouseLocationColumn(double x) {
         return (int) ((x / stepSize) + xDifference);
     }
+
+    public int mouseLocationColumn2(double x) {
```
Comment (Jipje) [inline src/main/java/gui/GraphDrawer.java]: Maybe give this method a better name.

File: src/main/java/gui/MenuController.java (hunk 1 above)
Comment (JaspervanTilburg) [inline src/main/java/gui/MenuController.java]: I uncommented it again

**Comment by JaspervanTilburg** [comment]
The panning now works with the go to node button and the bookmarks

**Comment by Jipje** [comment]
Awesome, if Marc and Eric approve when regarding their branch and certain conflicts perhaps. I say we merge.

**Pull request merged**